onnxruntime/core/providers/cloud/endpoint_invoker.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <memory>
+#include "gsl/gsl"
+#include "core/framework/tensor.h"
+#include "core/common/inlined_containers.h"
+
+namespace onnxruntime {
+namespace cloud {
+
+using EndPointConfig = std::unordered_map<std::string, std::string>;
+using TensorPtr = std::unique_ptr<onnxruntime::Tensor>;
+using TensorPtrArray = onnxruntime::InlinedVector<TensorPtr>;
+using ConstTensorPtrArray = gsl::span<onnxruntime::Tensor* const>;
+
+class EndPointInvoker {
```

Comment:
>EndPointInvoker [](http://example.com/codeflow?start=6&length=15)

Should we move the entire invoker things to common utility? If we do this to executor then we should also do it to invokers.